trusted-committer/es/06-advocating-for-the-communitys-needs-es.asciidoc

```diff
@@ -1,35 +1,35 @@
 [[advocating]]
 
-== Abogando por las necesidades de la comunidad
+== Abogar por las necesidades de la comunidad
 
-Las comunidades InnerSource existen en un contexto corporativo, por ende son más apretadas que comunidades Open Source.
-A veces los interes unitarios de la empresa están en desacuerdo con los de la comunidad.
+Las comunidades InnerSource existen en un contexto corporativo, por ende están más apretadas que las comunidades Open Source.
+A veces los interes de la unidad de negocio están en desacuerdo con los de la comunidad.
 Los Trusted Committers toman una perspectiva a largo plazo de su proyecto.
 Ellos entienden que un comunidad sana es una precondición para un código sano.
-Es por esto que muchas iniciativas de InnerSource están modeladas de la manera Apache, su lema es http://theapacheway.com/community-over-code/["Comunidad sobre código"].
-Unidades de la empresa, por otro lado, están naturalmente mas preocupadas por los productos producidos por una comunidad InnerSource.
-Ellos prefieren ver resultados a corto o mediano plazo que ayudan al resultado final.
+Es por esto que muchas iniciativas de InnerSource están modeladas de la manera Apache, modelados a la manera http://theapacheway.com/community-over-code/["Comunidad sobre código"].
```

Comment:
_modeladas **a** la manera Apache, su lema es..._ The original comment just meant to fix the preposition.